tensorflow/python/ops/nn_impl.py

```diff
@@ -1396,7 +1396,7 @@ def fused_batch_norm(
   min_epsilon = 1.001e-5
   epsilon = epsilon if epsilon > min_epsilon else min_epsilon
 
-  if compat.forward_compatible(2019, 6, 6):
+  if compat.forward_compatible(2019, 10, 10):
```

Comment:
Please change this date to something past Nov 11 as that is the 1.14 horizon: https://github.com/tensorflow/tensorflow/pull/29812